lib/LaTeXML/Package/algorithmic.sty.ltxml

```diff
@@ -15,6 +15,10 @@ use strict;
 use warnings;
 use LaTeXML::Package;
 
+# Was algorithmicx.sty loaded? If so: BAIL immediately.
+# (deeply incompatible)
+return 1 if IsDefined(T_CS('\algorithmic'));
```

Comment:
Hmm, didn't know you could ```return``` here!  This actually sounds like a reasonable approach after all.  But it probably would be better to issue an Error (or Warning), because otherwise the behaviour may be baffling. (ditto for algorithmicx)